homework1/1.1/1.1/1.1.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace Sort
```

Comment:
Используйте file-scoped namespaces из C# 10:
```suggestion
﻿namespace Sort;
```

И дальше без фигурной скобки и отступа